README.md

```diff
@@ -128,12 +125,96 @@ Invokes all Unit tests from schema/package "hr_test" with two reporters:
 * ut_coverage_html_reporter - will report only on database objects that are mapping to file structure from "source" folder and save output to file "coverage.html"
 
 ```
-utplsql run hr/hr@xe
+> utplsql run hr/hr@xe
 ```
 
 Invokes all unit test suites from schema "hr". Results are displayed to screen using default ut_documentation_reporter.
 
-#### Enabling Color Outputs on Windows
+### info
+`utplsql info [<ConnectionURL>]`
```

Comment:
Maybe make a separate section for the connection URL and link to it in each command. That way it is defined only once.

Reply:
Yes, came to my mind but wasn't able to come up with a good way and therefore decided to refactor later ;)